pkgs/tools/misc/dialog/default.nix

```diff
@@ -3,25 +3,30 @@
 , fetchurl
 , libtool
 , ncurses
-, withLibrary ? false
-, unicodeSupport ? true
 , enableShared ? !stdenv.isDarwin
+, unicodeSupport ? true
+, withLibrary ? false
 }:
 
-assert withLibrary -> libtool != null;
-assert unicodeSupport -> ncurses != null && ncurses.unicodeSupport;
+assert unicodeSupport -> ncurses.unicodeSupport;
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "dialog";
-  version = "1.3-20220728";
+  version = "1.3-20230209";
 
   src = fetchurl {
-    url = "ftp://ftp.invisible-island.net/dialog/dialog-${finalAttrs.version}.tgz";
-    hash = "sha256-VEGJc9VZpGGwBpX6/mjfYvK8c9UGtDaCHXfKPfRUGQs=";
+    urls = [
+      "https://invisible-island.net/archives/dialog/dialog-${finalAttrs.version}.tgz"
+      "ftp://ftp.invisible-island.net/dialog/dialog-${finalAttrs.version}.tgz"
```

Comment:
We shouldn't use ftp as a fallback, it is very insecure.

Reply:
It was ftp originally. I will remove it.